documentation/src/main/asciidoc/reference/introduction.adoc

```diff
@@ -0,0 +1,517 @@
+[[getting-started]]
+= Introduction to Hibernate Reactive
+
+:example: https://github.com/hibernate/hibernate-reactive/tree/master/example
+
+Before you start, we recommend taking a quick look at the example
+program in the {example}[`example`] directory, which shows off all
+the "bits" you'll need to get your own program up and running.
+
+== Setting up a reactive Hibernate project
+
+Setting up a project that uses Hibernate Reactive isn't hard at all.
+
+If you're using Hibernate Reactive outside of the Quarkus environment,
+you'll need to:
+
+- include Hibernate Reactive itself along with the appropriate Vert.x
+  reactive database client as dependencies of your project, and
+- configure Hibernate Reactive with information about your database,
+  using Hibernate configuration properties.
+
+Or, if you're using Hibernate Reactive in Quarkus.... TODO
+
+=== Including Hibernate Reactive in your project build
+
+:build: https://github.com/hibernate/hibernate-reactive/blob/master/example/build.gradle
+:slf4j: http://www.slf4j.org/
+
+Add the following dependency to your project:
+
+----
+org.hibernate.reactive:hibernate-reactive-core:{version}
+----
+
+Where `{version}` is the version of Hibernate Reactive you're using.
+
+You'll also need to add a dependency for the Vert.x reactive database
+driver for your database, for example:
+
+- `io.vertx:vertx-pg-client` for Postgres,
+- `io.vertx:vertx-mysql-client` for MySQL, or
+- `io.vertx:vertx-db2-client` for DB2.
+
+You don't need to depend on the JDBC driver for your database (but you
+can if you like).
```

Comment:
This sentence sounds weird (why somebody would add it if it's not needed?), maybe you can clarify what's the difference with having the JDBC driver on the classpath.

Reply:
Because this is  an introduction, I'm fine with removing this section from here

Reply:
Or we can just remove this sentence